domains/deekshasharma.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio",
+  "owner": {
+    "username": "Deeksha2501",
+    "email": "sharmadeeksha325@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://deeksha2501.github.io/PORTFOLIO/",
+    "URL": "http://deekshasharma.vercel.app/"
```

CNAME records can only be host names and you cant have both a URL redirection and a CNAME subdomain.

If you meant to use a subdomain, this should be enough
```suggestion
    "CNAME": "deeksha2501.github.io"
```